Source/Plugins/Core/com.equella.core/resources/web/scripts/courselist.js

```diff
@@ -63,115 +63,148 @@ var CourseList = {
 		var itemResultSelector = '.selectable.' + itemResultClass;
 		var itemSummarySelector = "h2." + itemSummaryClass;
 		var attachmentSelector = '.selectable .attachmentrow.with-select';
+		var newSearchPageItemSelector = '.NewSearchPage-Item';
+		var newSearchPageAttachmentSelector = '.NewSearchPage-Attachment';
```

Comment:
This is a super minor thing - so feel free to ignore - but, do we really need to prefix the classes (which end up in the DOM and on the React side) with `New`. Could these just be:

```suggestion
		var newSearchPageItemSelector = '.SearchPage-Item';
		var newSearchPageAttachmentSelector = '.SearchPage-Attachment';
```